.github/workflows/pssanalyzer.yml

```
@@ -0,0 +1,12 @@
+name: Run PSScriptAnalyzer
+
+on: [push, pull_request]
```

While not very likely to be used, let's allow the workflow to run manually:

```suggestion
on: [push, pull_request, workflow_dispatch]
```